src/legacy/ui/field_formats/converters/custom.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FieldFormat } from '../field_format';
+import { FieldFormatConvert } from '../types';
+
+const ID = 'custom';
+
+export const createCustomFieldFormat = (convert: FieldFormatConvert) =>
```

Comment:
This is used only once, and it's a bit unclear when read out of the context of `FieldFormat.from`.
Do you think we could somehow just make this a private method of `FieldFormat`?
If not, this could use a  short explanation why is it here \ where it's used..

Reply:
My idea was to make it similar to other formatters. See `src\legacy\core_plugins\kibana\common\field_formats\types\ `folder.  I don't like an idea to make it **private**. I think it will be better in future to delete the FieldFormat.from method and use `createCustomFieldFormat` instead.